build04/configuration.nix

```diff
@@ -3,15 +3,15 @@
   imports = [
     inputs.disko.nixosModules.disko
     ./hardware-configuration.nix
-    ../roles/common.nix
-    ../roles/hercules-ci
-    ../roles/remote-builder/user.nix
+    inputs.self.modules.nixos.common
```

Comment:
Might be worth it to also pass self as a specialArg and use that directly.

Reply:
I'll leave this for now as we need inputs in a few places anyway.